repondeur/zam_repondeur/views/lectures.py

```diff
@@ -4,13 +4,13 @@
 from datetime import datetime
 from typing import BinaryIO, Dict, Iterable, TextIO, Tuple
 
-from pyramid.httpexceptions import HTTPFound, HTTPNotFound
+from pyramid.httpexceptions import HTTPBadRequest, HTTPFound, HTTPNotFound
 from pyramid.request import Request
 from pyramid.response import Response
 from pyramid.view import view_config, view_defaults
 from sqlalchemy.sql.expression import case
 
-from zam_aspirateur.textes.models import Chambre
+from zam_aspirateur.textes.models import Chambre, Dossier, Lecture as LectureData
```

Comment:
We use to call the one from repondeur `FooModel`, at least in `fetch.py`, uniformize?